spark-operator-api/src/main/java/org/apache/spark/k8s/operator/Constants.java

```diff
@@ -80,4 +84,12 @@ public class Constants {
       "The Spark application is running with less than minimal number of requested executors.";
   public static final String EXECUTOR_LAUNCH_TIMEOUT_MESSAGE =
       "The Spark application failed to get enough executors in the given time threshold.";
+
+  // Spark Cluster Messages
+  public static final String CLUSTER_SCHEDULE_FAILURE_MESSAGE =
+      "Failed to request Spark cluster from scheduler backend.";
+  public static final String CLUSTER_SUBMITTED_STATE_MESSAGE =
+      "Spark cluster has been created on Kubernetes Cluster.";
```

Comment:
"Submitted" doesn't mean the cluster is created, right? Should it be "submitted?